src/styles/utils/typography.ts

```diff
@@ -106,6 +106,7 @@ export type TypographyProps = FontProps & {
   $wordWrap?: ResponsiveValues<"normal" | "break-word" | "initial" | "inherit">;
   $textOverflow?: ResponsiveValues<"clip" | "ellipsis">;
   $overflowWrap?: ResponsiveValues<"normal" | "break-word" | "anywhere">;
+  $fontWeight?: ResponsiveValues<FontWeight>;
```

Comment:
is this not already within the `FontProps`?

Reply:
I think font weight is tied into the 'font' types. 'body-1' etc - I think this was to keep it restricted to the predefined font types. Could a font type be passed into OutlineHeading ?

Reply:
Ah I see, makes sense! Yeah I agree then we should pass in a font type - if there is no predefined font type that fits we either need to make one or adjust the design to use a predefined one (I'd prefer the latter)

Reply:
I did this because when I passed a font into the outline heading the text below would assume the position underneath the heading
![Screenshot 2024-01-10 at 09 59 03](https://github.com/oaknational/Oak-Web-Application/assets/91190841/0efb855e-c5dd-4182-99c3-7473e4aed567)

Will try an alternative solution